msteams-platform/concepts/resource-specific-consent.md

```diff
@@ -0,0 +1,90 @@
+
+1. Register your application in Azure AD.
+2. Validate permissions.
+3. Update your Teams app manifest to link to your AAD app ID.
+4. Get an access token.
+5. Make a Graph call.
+
+
+#### 1. Register a new application in Azure AD
+
+[Register](~/graph/auth-register-app-v2) a new application in the [Azure Active Directory- App Registration portal](https://ms.portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade) to get an AAD app ID. If your AAD app supports [single sign-on (SSO) for tabs](~/microsoftteams/platform/tabs/how-to/authentication/auth-aad-sso), then use the same AAD app ID for configuring RSC.
+
+#### 2. Validate permissions
+* If you are configuring RSC with a newly registered app (with default permissions): Follow step 3. 
+* If you are configuring RSC with an existing app (with additional permissions): Check the permissions in the `API Permissions` section of the app registration portal in Azure AD, and confirm if RSC related permissions are configured here. If so, they should be deleted from here and are supposed to be configured in the Teams app manifest as mentioned in step 3.
+
+#### 3. Update your Teams app manifest to link to your AAD app ID
+Add a [webApplicationInfo](~/microsoftteams/platform/resources/schema/manifest-schema#webapplicationinfo) section to the manifest that includes two properties - `id` is the AAD app ID, and `applicationPermissions` are the RSC permissions (scopes) your app requires.
+
+> **Note**: If your app needs non RSC permissions, you can continue to use them as always, but they do not go in the manifest. 
+
+```
+{ 
+  "$schema": ".../MicrosoftTeams.schema.json", 
+   "webApplicationInfo": { 
+      "id": "cb38cf54-ac89-4a7a-9ea3-095d3d080037", 
+      "applicationPermissions": [  
+      "File.Read.Group",
+      "TeamSettings.Read.Group",
+      "ChannelMessage.Read.Group",
+      "Member.Read.Group"
+      ] 
+  }
+  ... 
+} 
+```
+Refer to the list of available Graph permissions for RSC in [Teams permissions](~/graph/permissions-reference.md) section.
+
+#### 4. Get an access token 
+
+Before you make a REST call to the Graph, you need to [get an access token](~/graph/auth-v2-service) for the application permissions similar to getting an application permission token for non-RSC use.
+  
+```
+string response = await HttpHelpers.POST( 
+    $"https://login.microsoftonline.com/{tenant}/oauth2/v2.0/token", 
+    $"grant_type=client_credentials&client_id={appId}&client_secret={appSecret}" 
+    + "&scope=https%3A%2F%2Fgraph.microsoft.com%2F.default"); 
+string token = response.Deserialize<TokenResponse>().access_token; 
+```
+  
+#### 5. Make a Graph call 
+
+You can now make a REST call similar to a non RSC call or use the SDK. See [Microsoft Graph](https://developer.microsoft.com/en-us/graph) and [Use the Microsoft Graph API to work with Microsoft Teams](~/graph/api/resources/teams-api-overview?view=graph-rest-beta) for more details. 
+
+##  RSC Guidelines
```

Comment:
There's gotta be a better place in this doc to put this information – this is just a grab bag of random info.

Reply:
Moved these blocks to the relevant sections of the document.